.github/workflows/compatibility.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         woocommerce: [ 'beta' ]
         wordpress:   [ 'latest' ]
         gutenberg:   [ 'latest' ]
-        php:         [ '7.1', '8.0', '8.1' ]
+        php:         [ '7.2', '8.0', '8.1' ]
```

Comment:
Should we update to PHP `7.4` since it's the least supported version based on https://www.php.net/supported-versions.php